tests/integration/test_docker_manager_integration.py

```diff
@@ -39,19 +36,27 @@
 
 pytestmark = pytest.mark.anyio
 
+# Path to test fixtures
+FIXTURES_DIR = Path(__file__).parent / "fixtures"
```

Comment:
I think we should use `resources.files` from `importlib import resources`, i.e., use `resources.files("tests.integration.fixtures")`.

This returns a Path-like object on which we can then call `str` later on